config/jest.setup.js

```diff
@@ -7,8 +7,25 @@ require('whatwg-fetch');
 
 require('../app/assets/javascripts/miq_global');
 
+window.add_flash = (message, type) => {
```

Comment:
#5027 introduced helpers for these functions, we should probably just require those in the specs that need them